app/models/spree/taxon.rb

```diff
@@ -91,9 +85,7 @@ def self.distributed_taxons(which_taxons = :all)
     private
 
     def permalink_end
-      return UrlGenerator.to_url(name) if permalink.blank?
-
-      permalink.split('/').last
+      permalink.blank? ? UrlGenerator.to_url(name) : permalink.split('/').last
```

Comment:
Why did you change this? Ternary operators are hard to read.

Reply:
I'll revert this as well